nodebuilder/p2p/flags.go

```diff
@@ -92,7 +92,7 @@ func ParseNetwork(cmd *cobra.Command) (Network, error) {
 func parseNetworkFromEnv() (Network, error) {
 	var network Network
 	// check if custom network option set
-	// format: CELESTIA_CUSTOM=<netID>:<genesisHash>:<bootstrapPeerList>
+	// format: CELESTIA_CUSTOM=<netID>:<genesisBlockHash>:<bootstrapPeerList>
```

Comment:
Can the node not query this from the consensus network via:

consensus-node:26657/status - "earliest_block_hash"

Reply:
It can, if we wanna move on making node always dependent on core which wasn't the case.